craft_providers/actions/snap_installer.py

```diff
@@ -100,7 +95,7 @@ def _get_host_snap_revision(snap_name: str) -> str:
             brief="Unable to connect to snapd service."
         ) from error
     snap_info.raise_for_status()
-    return snap_info.json()["result"]["revision"]
+    return snap_info.json()["result"]
```

Comment:
it might be beneficial to have this result represented in a dataclass; todo for later

Reply:
Added as a TODO